src/__tests__/Carousel.tsx

```diff
@@ -189,6 +189,15 @@ describe('Slider', function() {
                 });
             });
         });
+
+        describe('Different child types', () => {
```

Comment:
This test isn't good as it doesn't assert anything, but was mainly added as a way to quickly see the typescript errors from #703